stl/src/locale0.cpp

```diff
@@ -3,6 +3,8 @@
 
 // class locale basic member functions
 
+#undef _ENFORCE_ONLY_CORE_HEADERS // TRANSITION, <xfacet> should be a core header

```

Comment:
Can/should we do it more precisely, like this:
```c++
#pragma push_macro(_ENFORCE_ONLY_CORE_HEADERS)
#undef _ENFORCE_ONLY_CORE_HEADERS // TRANSITION, <xfacet> should be a core header
#include <xfacet>
#pragma pop_macro(_ENFORCE_ONLY_CORE_HEADERS)
```

Reply:
I don't think that would achieve anything useful. This whole file is the scariest single file in the entire STL. We shouldn't be messing with it, so making the exemption fine-grained seems like unnecessary complexity.